executor/adapter.go

```diff
@@ -218,16 +215,8 @@ func (a *ExecStmt) Exec(ctx context.Context) (ast.RecordSet, error) {
 	var pi processinfoSetter
 	if raw, ok := sctx.(processinfoSetter); ok {
 		pi = raw
-		sql := a.OriginText()
-		if simple, ok := a.Plan.(*plannercore.Simple); ok && simple.Statement != nil {
-			if ss, ok := simple.Statement.(ast.SensitiveStmtNode); ok {
-				// Use SecureText to avoid leak password information.
-				sql = ss.SecureText()
```

Comment:
Is the password protected now?

Reply:
I think it's important not to leak user's information.
Could we remove `cc.ctx.SetProcessInfo("", t, cmd)` in `dispatch` and do it here?
```
pi.SetProcessInfo(sql, t, CmdQuery)
```
@imtbkcat